CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+- **Breaking**: change ``cartridge pack`` naming strategy:
+  * RPM: `<app-name>-<version>[.<suffix>]-1.<arch>.rpm`,
+  * DEB: `<app-name>-<version>[.<suffix>]-1_<arch>.deb`,
+  * TGZ: `<app-name>-<version>[.<suffix>].tar.gz`.
```

Comment:
TGZ is not for sources, right? So we should have an architecture here as well?

Reply:
I'm a bit stressed out about this one since we no longer will be able to set a TGZ name with `--version`to be **exactly** equal to pre-patch one, but the point seems fair. Added it.

Reply:
1. It seems, it is not reflected in the changelog (for tgz).
2. AFAIS, Debian package has 'all' architecture.

Reply:
1. Fixed
2. What's the question? To replace <arch> with specific values since for now they all are constants?

Reply:
Closed after live discussion